### ISSUE TITLE
All apps have the same priority; pressing Up crashes the app

### ISSUE THREAD

**Issue by RavenMacDaddy** [Codectory/AutoHDR#57]
![image](https://user-images.githubusercontent.com/67732686/144752829-5b6b2a59-757f-4e8a-95f4-74ff97d204f5.png)

Pressing down does nothing.

Pressing up crashes the app.

**Comment by Codectory** [comment]
Oops, it was like that on start up?

RavenMacDaddy ***@***.***> schrieb am So. 5. Dez. 2021 um
16:26:

**Comment by RavenMacDaddy** [comment]
Yep, crashes when pressing up and maybe down as well, maybe depending on what numbering is present, but most of the time they're all the same priority which seems to be based on the amount of apps in the list.

**Comment by Codectory** [comment]
Could you please manually edit the usersettings.json and change the
position of each app?

RavenMacDaddy ***@***.***> schrieb am So. 5. Dez. 2021 um
18:04:

**Comment by RaasclartNinja** [comment]
I've manually edited the .json and set the priorities in order 0-8 to try and sort the list but it doesn't change the order.

![image](https://user-images.githubusercontent.com/49550861/144757955-7c441774-faad-418b-87b1-b7034fac8bb1.png)

**Comment by Codectory** [comment]
Please send me the json file. I cannot reproduce this.

**Comment by RaasclartNinja** [comment]
Sure, here you 
[UserSettings.zip](https://github.com/Codectory/AutoHDR/files/7656626/UserSettings.zip)
go

**Comment by RavenMacDaddy** [comment]
Editing the json as suggested, simply giving every app their own priority and then using the arrows to create the desired order worked for me - though I wonder what caused the priorities to become the same in the first place.

I upgrade the app by simply overwriting with the files from the archive - is that a risk of some kind?

**Comment by RaasclartNinja** [comment]
For me the Up/Down buttons don't sort this list in anyway that I can make sense of, it almost seems random. I cannot get Age of Empires IV to the top as an example.

**Comment by Codectory** [comment]
Should be fixed in 1.7.11

**Comment by RaasclartNinja** [comment]
That's awesome, thanks for the hard work!
